lib/src/Uncertainty/StatTests/VisualTest.cxx

```diff
@@ -343,7 +343,7 @@ GridLayout VisualTest::DrawPairsMarginals(const Sample & sample, const Distribut
   const Description description(sample.getDescription());
   for (UnsignedInteger i = 0; i < dimension; ++ i)
   {
-    Graph pdfGraph(distribution.getMarginal(i).drawPDF());
+    Graph pdfGraph(distribution.getMarginal(i).drawPDF(sample.getMarginal(i).getMin()[0], sample.getMarginal(i).getMax()[0]));
```

Comment:
I would rather use a small margin to ensure that the data is safely shown.

Reply:
or maybe the other way: as drawPDF() produces a quite nice graph, we could use it to present the clouds.

Reply:
I think the clouds would be quite narrow if you look at the image from the issue

Reply:
Thanks for your comments. I tried to implement your solution in C++.

This is the result

![image](https://github.com/user-attachments/assets/9ba8b004-ef3c-49ed-9f64-d73c15d0f31b)